### ISSUE TITLE
Error Running Analysis after creating Fermipy environment with Miniforge

### ISSUE THREAD

**Issue by bandarr5001** [fermiPy/fermipy#626]
When creating the fermipy environment with miniforge, the command conda list -f fermitools will fail in the file __init__.py when trying to run the data analysis. This results in permission error (permission denied: 'conda').

The current workaround I am using (provided by Niccolò Di Lalla) is opening the file at:
/home/bandarr/miniforge3/envs/fermi/lib/python3.9/site-packages/fermipy/__init__.py
and at line 59 (above the try/except) adding:
return '2.2.0'
or whatever version of fermitools is installed in your environment.

**Comment by HealthyPear** [comment]
If it can help I followed the instructions here
https://fermipy.readthedocs.io/en/latest/install.html#conda-based-installation

and I never encountered this issue 

my setup is the following (for both macos 12 and 14):

mamba 2.0.5

fermipy          1.3.1    pypi_0          pypi
fermitools       2.2.0    py39h5f0296b_0  fermi
fermitools-data  0.18     0               fermi

**Comment by bandarr5001** [comment]
Yeah I am also following those instructions, I'm on Windows Subsystem for Linux though.

**Comment by ndilalla** [comment]
The handling of `fermitools` version has been improved in `fermipy 1.4.1` and should no longer be affected by this issue. I am closing this issue, feel free to open it again if this is still a problem.
